images-list

```diff
@@ -984,6 +986,7 @@ quay.io/prometheus/prometheus rancher/mirrored-prometheus-prometheus v2.38.0
 quay.io/prometheus/prometheus rancher/mirrored-prom-prometheus v2.18.2
 quay.io/prometheus/prometheus rancher/prom-prometheus v2.19.2
 quay.io/prometheus/prometheus rancher/prom-prometheus v2.22.0
+quay.io/skopeo/stable rancher/mirrored-skopeo-skopeo v1.10
```

Comment:
This seems to be a "moving tag", for the latest v1.10.x, meaning it will mirror on merge for the current one marked as v1.10.x and then again whenever something else is merged. But if nothing is merged for weeks, it won't be up-to-date with quay.io.

I also don't think we have another image in this file that doesn't have the explicit version tag.

I'm just checking here, if you want/need this to happen this way, let me know.

Reply:
I will talk with Enrico. I do not believe that a moving tag is desired for integration. See paketobuilder, where we pin to a specific version as well.

Reply:
Will be changed and pinned to `v1.10.0`.

Reply:
Done, commit e1b4faf6e3ba4fd92a1218c088173a99879f752d